coderd/metrics.go

```diff
@@ -0,0 +1,177 @@
+package coderd
+
+import (
+	"encoding/json"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/google/uuid"
+	"nhooyr.io/websocket"
+	"nhooyr.io/websocket/wsjson"
+
+	"cdr.dev/slog"
+	"github.com/coder/coder/coderd/database"
+	"github.com/coder/coder/coderd/httpapi"
+	"github.com/coder/coder/coderd/httpmw"
+	"github.com/coder/coder/coderd/rbac"
+	"github.com/coder/coder/codersdk"
+)
+
+const AgentStatIntervalEnv = "CODER_AGENT_STAT_INTERVAL"
```

Comment:
Indicate the units??
```suggestion
const AgentStatIntervalEnv = "CODER_AGENT_STAT_INTERVAL_MS"
```